### ISSUE TITLE
"Terms" and "Privacy" links are doesn't work

### ISSUE THREAD

**Issue by CAMOBAP** [hCaptcha/HCaptcha-ios-sdk#114]
## Bug description

`Terms` or `Privacy` links should lead to corresponding pages

## To Reproduce

`hcaptcha.validate` and try to click `Terms` or `Privacy` links

## Expected behavior

`Terms` or `Privacy` links should lead to corresponding pages

## Additional context

It works on Android

**Comment by e271828-** [comment]
You're right, this used to work so I'm not sure when it was changed. It still works for the accessibility link, so I think you just need to adjust the filter as to what's allowed to open an external link.

**Comment by CAMOBAP** [comment]
@e271828- it works for the accessibility link because it contains `target="_black"` Unfortunately there is no such attribute + listener in JS, so Swift haven't a chance to intercept it in 

https://github.com/hCaptcha/HCaptcha-ios-sdk/blob/master/HCaptcha/Classes/HCaptchaWebViewManager.swift#L364-L370

**Comment by e271828-** [comment]
This has been resolved upstream
